libs/sdk-react-native/ios/BreezSDKMapper.swift

```diff
@@ -681,7 +686,8 @@ class BreezSDKMapper {
             "feesHash": reverseSwapPairInfo.feesHash,
             "feesPercentage": reverseSwapPairInfo.feesPercentage,
             "feesLockup": reverseSwapPairInfo.feesLockup,
-            "feesClaim": reverseSwapPairInfo.feesClaim
+            "feesClaim": reverseSwapPairInfo.feesClaim,
+            "feesTotalEstimated": reverseSwapPairInfo.feesTotalEstimated
```

Comment:
Needs renaming to `totalEstimatedFees`

Reply:
Done in f02c462fb1406ad4f07a94a1a9dfb440e140bec1